src/NzbDrone.Core.Test/Localization/LocalizationServiceFixture.cs

```diff
@@ -32,35 +32,28 @@ public void should_get_string_in_dictionary_if_lang_exists_and_string_exists()
         [Test]
         public void should_get_string_in_default_language_dictionary_if_no_lang_country_code_exists_and_string_exists()
```

Comment:
Is the test name still relevant?

Reply:
I don't know if it was ever relevant, but updated it it to the bare minimum, getting the translation in French.